bin/reth/src/commands/test_vectors/tables.rs

```diff
@@ -81,7 +81,7 @@ where
 {
     let mut rows = vec![];
     let mut seen_keys = HashSet::new();
-    let strat = proptest::collection::vec(
+    let start = proptest::collection::vec(
```

Comment:
i don't think this is correct, i think this is strat as in strategy?

Reply:
thanks for that, renamed to strategy instead.